src/test/java/io/lighty/yang/validator/check_update_from/RFC6020Test.java

```diff
@@ -5,7 +5,7 @@
  * terms of the Eclipse Public License v1.0 which accompanies this distribution,
  * and is available at https://www.eclipse.org/legal/epl-v10.html
  */
-package io.lighty.yang.validator.checkUpdateFrom;
+package io.lighty.yang.validator.check_update_from;
```

Comment:
using underscore in java packages is not a good practice (there are just few exceptions when it should be used but those are not applicable here)
please do not use underscores in package names

Reply:
Done